configs/components/puppet.rb

```diff
@@ -8,6 +8,16 @@
   if platform.is_windows?
     pkg.build_requires "rubygem-win32-dir"
   end
+  # Used to compile binary translation files
+  if platform.is_osx?
+    pkg.build_requires "gettext"
+  elsif platform.is_windows?
+    pkg.build_requires "pl-gettext-#{platform.architecture}"
+  elsif platform.is_aix?
+    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-gettext-0.19.8-2.aix#{platform.os_version}.ppc.rpm"
```

Comment:
This is cool. We don't officially support it on AIX, so if i18n suddenly breaks, does this commit commit us (heh, see what I did there?) to unofficially supporting i18n on AIX?